PuzzleServer.sln

```diff
@@ -21,6 +21,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		.editorconfig = .editorconfig
 	EndProjectSection
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "PuzzleSafari", "PuzzleSafari\PuzzleSafari.csproj", "{87954399-5E59-47C9-B63B-A16C94C8D47D}"
```

Comment:
remove this stuff if/when you remove the PuzzleSafari project

Reply:
Reverted.

---
In reply to: [295581803](https://github.com/PuzzleServer/mainpuzzleserver/pull/491#discussion_r295581803) [](ancestors = 295581803)